src/models/bucket.ts

```diff
@@ -23,15 +29,16 @@ export async function findInstance(batch: number): Promise<string | undefined> {
 }
 
 async function updateInstanceCache(batch: number): Promise<void> {
+  const bucket = STAGING_BUCKET;
```

Comment:
If you always use STAGING here, isn't the link to the instance not going to work anymore?

Reply:
This is just for the instance files. We always use staging standard solver instance file for the link. Agreed that this is not ideal though since things like token filters and configured parameters may differ from instance to instance and the link should probably be updated to match the solver's actual instance in case of a solution. I do think this is a separate task though.